colonyShip_test.go

```diff
@@ -10,4 +10,6 @@ func TestColonyShipSpeed(t *testing.T) {
 	cs := newColonyShip()
 	speed := cs.GetSpeed(Researches{ImpulseDrive: 6})
 	assert.Equal(t, 5500, speed)
+	assert.Equal(t, 8700, cs.GetCargoCapacity(Researches{HyperspaceTechnology: 8}))
```

Comment:
This is definitely not a testing Speed :P
You should create a test for CargoCapacity exclusively.